tensorboard/components/experimental/plugin_lib/polymer-interop.ts

```diff
@@ -0,0 +1,29 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+import * as publicApi from './index';
+
+namespace tb_plugin_lib.experimental {
+  export const core = publicApi.core;
+  export const runs = publicApi.runs;
+}
```

Comment:
Would we want to carefully select which APIs to expose, or should we just wholesale export:
```
namespace tb_plugin_lib {
    experimental: publicApi
}
```
?